src/gtk/webview_webkit2.cpp

```diff
@@ -608,6 +682,80 @@ wxgtk_authorize_authenticated_peer_cb(GDBusAuthObserver *,
 
 } // extern "C"
 
+//-----------------------------------------------------------------------------
+// wxWebViewConfigurationImplWebKit
+//-----------------------------------------------------------------------------
+class wxWebViewConfigurationImplWebKit : public wxWebViewConfigurationImpl
+{
+public:
+    wxWebViewConfigurationImplWebKit():
+        m_webContext(nullptr)
+#if WEBKIT_CHECK_VERSION(2, 10, 0)
```

Comment:
Should we perhaps define some `HAVE_WEBKIT_WEBSITE_DATA_MANAGER` once instead of checking for the version multiple times? This would be a bit more readable, IMHO, and would make it easier to get rid of this check later when we drop support for the versions < 2.10.